### DISCUSSION TITLE
What is that feature?

### DISCUSSION THREAD

**Discussion by Folleach** [Folleach/GeometryDashAPI#11] [General]
I just want to more about it feature

**Comment by Folleach** [comment]
Well, looks interesting

**Comment by Folleach** [comment]
But why it use "global id" #11?
I don't want them to get clogged, it isn't better then `issues`

**Comment by ItzSkyReed** [comment]
Hello

**Reply by Folleach** [reply]
Hello )

**Comment by Folleach** [comment]
Okay, it was a test discussion, close
